packages/website/docs/crawler.mdx

```diff
@@ -98,6 +98,14 @@ We provide [config templates](/docs/templates) for many website generators, but
 
 The Crawler only applies `index settings` at index creation index, to keep the Algolia dashboard as the source of truth. If you have drastically changed your config, or moved to a website generator, we recommend you to delete your index from the Algolia dashboard before starting a new crawl.
 
+## Can I delete my crawler?
+
+No. Well, you can but once you do things will not work correctly. We automatically create a default crawler that is associated with your DocSearch application and deleting it with the intention of creating a new one will not work as expected.
+
+## What if I delete my DocSearch Crawler?
+
+The fastest way will be to connect with us on our Discord server. Alternatively, email us at the address below and we will get to it as soon as we can.
```

Comment:
Maybe put the discord link again here

Reply:
ah yeah! I meant to do that and forgot. thanks!